Source/WebKit/DerivedSources.make

```diff
@@ -599,6 +599,7 @@ SERIALIZATION_DESCRIPTION_FILES = \
 	Shared/Cocoa/SharedCARingBuffer.serialization.in \
 	Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in \
 	Shared/CallbackID.serialization.in \
+    Shared/ContentWorldData.serialization.in \
```

Comment:
Nit - looks like this uses spaces while the other lines use tabs.

Reply:
Fixed.